### PR TITLE
Completed css fonts exercise.

### PR DESCRIPTION
**Pull request by randygerada**
css-fonts-merri
Screen Shot 2022-10-12 at 2 26 13 PM

### CODE REVIEW

File: css-fonts/styles.css [hunk 1]
```diff
@@ -0,0 +1,8 @@
+p {
+  font-family: 'Merriweather', serif;
+  font-family: 'Dosis', sans-serif;
```
Comment (BrettAlbright) [inline css-fonts/styles.css:6]: you don't want to set two different font families on the same type of element, the second will override the first. I would apply one of these fonts to the `h1` and `h2` selectors

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed

**Comment by randygerada** [comment]
Screen Shot 2022-10-12 at 2 32 58 PM

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**